src/validations/signUpValidations.js

```diff
@@ -1,13 +1,28 @@
+import strings from 'localization';
+
+const { VALIDATIONS } = strings;
+
 export default {
+  firstName: {
+    presence: { message: VALIDATIONS.firstName.presence },
+  },
   email: {
     presence: true,
-    email: true,
+    email: { message: VALIDATIONS.email.email },
   },
   password: {
     presence: true,
+    length: {
+      minimum: 8,
+      message: VALIDATIONS.password.length,
+    },
   },
   passwordConfirmation: {
     presence: true,
-    equality: { attribute: 'password' },
+    equality: { attribute: 'password', message: VALIDATIONS.passwordConfirmation.equality },
```

Comment:
maybe as a readability improvement you can do 
```suggestion
    equality: { 
       attribute: 'password', 
       message: VALIDATIONS.passwordConfirmation.equality
   },
```